core-common/src/main/kotlin/com/twitter/rules/core/util/KtCallableDeclarations.kt

```diff
@@ -2,12 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 package com.twitter.rules.core.util
 
-import org.jetbrains.kotlin.psi.KtParameter
+import org.jetbrains.kotlin.psi.KtCallableDeclaration
 
-val KtParameter.isTypeMutable: Boolean
-    get() = typeReference?.text?.matchesAnyOf(KnownMutableCommonTypes) == true
+val KtCallableDeclaration.isTypeMutable: Boolean
+    get() = typeReference?.text?.matchesAnyOf(KnownMutableCommonTypesRegex) == true
 
-private val KnownMutableCommonTypes = sequenceOf(
+val KnownMutableCommonTypesRegex = sequenceOf(
     // Set
     "MutableSet<.*>\\??",
     "ArraySet<.*>\\??",
```

Comment:
What is someone is using `HashMap<>` or `ArrayList`, etc?

Reply:
Those are already caught by a different rule to catch inherently mutable parameters, `ComposeMutableParams`